common.eai

```diff
@@ -12568,41 +12553,40 @@ endfunction
 //============================================================================
 function SleepUntilTownDefended takes integer ai_strength returns nothing
   local integer defense_length_counter = 0
-  set own_strength = ai_strength
+  local real t = 2 * sleep_multiplier
```

Comment:
Not sure why you pulled this out as a local variable when we only use it once.

Reply:
Indeed, you changed `sleep_maultiplier` to dynamic before, I forgot about that